specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-03-01-preview/examples/CosmosDBManagedCassandraClusterListByResourceGroup.json

```diff
@@ -0,0 +1,76 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
+    "resourceGroupName": "cassandra-prod-rg",
+    "api-version": "2021-03-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/subid/resourceGroups/cassandra-prod-rg/providers/Microsoft.DocumentDB/cassandraClusters",
+            "name": "cassandra-prod",
+            "type": "Microsoft.DocumentDB/cassandraClusters",
+            "location": "West US",
+            "tags": {},
+            "properties": {
+              "provisioningState": "Succeeded",
+              "delegatedManagementSubnetId": "/subscriptions/536e130b-d7d6-4ac7-98a5-de20d69588d2/resourceGroups/customer-vnet-rg/providers/Microsoft.Network/virtualNetworks/customer-vnet/subnets/management",
```

Comment:
the term "Management" means nothing to the user. As far as they are concerned, this is the subnet where the ip address of the prometheus endpoint appears.

I recommend calling this "delegatedPrometheusSubnetId"

Reply:
After discussion within the group, we've decided not to change this, since prometheus will be optional but this is not, and we might end up adding another endpoint as well.